pubspec.yaml

```diff
@@ -16,7 +19,10 @@ dependencies:
   http_parser: ^4.0.2
   device_info_plus: ^10.1.0
   path_provider: ^2.0.15
-  sim_card_info: ^1.0.2
+  sim_card_info:
```

Comment:
@vaishnavi-2301  Add a comment here with links to the OG package to check for releases periodically to see if the author includes a change to fix this in their package, then we will stop maintaining our fork.

Reply:
done